src/components/ButtonNav/index.tsx

```diff
@@ -3,18 +3,33 @@ import { StyledButton, Wrapper, StyledLink } from './styled'
 import { UserEdit, Users } from '@styled-icons/fa-solid'
 import { PermContactCalendar } from '@styled-icons/material-rounded'
 
-const ButtonNav: React.FC = () => (
-	<Wrapper>
-		<StyledLink to="/profile">
-			<StyledButton icon={UserEdit} title="Můj profil" />
-		</StyledLink>
-		<StyledLink to="/search-matching">
-			<StyledButton icon={Users} title="Matching" />
-		</StyledLink>
-		<StyledLink to="/">
-			<StyledButton icon={PermContactCalendar} title="Kontakty" />
-		</StyledLink>
-	</Wrapper>
-)
+interface Props {
+	login: boolean
+}
 
-export default ButtonNav
+export const ButtonNav: React.FC<Props> = ({ login }) => {
+	const NavBeforeLogin = (
+		<>
+			<StyledLink to="/register">
+				<StyledButton icon={UserEdit} title="Registrace" />
+			</StyledLink>
+			<StyledLink to="/login">
+				<StyledButton icon={PermContactCalendar} title="Přihlášení" />
+			</StyledLink>
+		</>
+	)
+	const NavAfterLogin = (
+		<>
+			<StyledLink to="/profile">
+				<StyledButton icon={UserEdit} title="Můj profil" />
+			</StyledLink>
+			<StyledLink to="/matching">
+				<StyledButton icon={Users} title="Matching" />
+			</StyledLink>
+			<StyledLink to="/contacts">
+				<StyledButton icon={PermContactCalendar} title="Kontakty" />
+			</StyledLink>
+		</>
+	)
+	return <Wrapper>{login ? NavAfterLogin : NavBeforeLogin}</Wrapper>
+}
```

Comment:
This could be written more separately like so:
```suggestion
const NavBeforeLogin = (
	<Wrapper>
		<StyledLink to="/register">
			<StyledButton icon={UserEdit} title="Registrace" />
		</StyledLink>
		<StyledLink to="/login">
			<StyledButton icon={PermContactCalendar} title="Přihlášení" />
		</StyledLink>
	</Wrapper>
)

const NavAfterLogin = (
	<Wrapper>
		<StyledLink to="/profile">
			<StyledButton icon={UserEdit} title="Můj profil" />
		</StyledLink>
		<StyledLink to="/matching">
			<StyledButton icon={Users} title="Matching" />
		</StyledLink>
		<StyledLink to="/contacts">
			<StyledButton icon={PermContactCalendar} title="Kontakty" />
		</StyledLink>
	</Wrapper>
)

export const ButtonNav: React.FC<Props> = ({ login }) => login ? NavAfterLogin : NavBeforeLogin
```

Reply:
Good 👍 fixed 🙂